db/migrate/20211110193730_add_similarity_to_fikas.rb

```diff
@@ -1,7 +1,6 @@
 class AddSimilarityToFikas < ActiveRecord::Migration[6.1]
   def change
     add_column :fikas, :similarity, :float
-    add_column :participants, :department, :string
     add_column :participants, :seniority, :integer
```

Comment:
In the discussion we talked about having seniority as boolean?

Reply:
We will discuss this as a team and make a chore for any changes decided